### PR TITLE
increasing the max line length for python tools checkstyle

### PR DESCRIPTION
**Pull request by tulinkry**
Increase the line length in python tools to 120

### CODE REVIEW

File: opengrok-tools/pom.xml [hunk 1]
```diff
@@ -280,6 +280,8 @@ Portions Copyright (c) 2017-2018, 2020, Chris Fraire <cfraire@me.com>.
                             <arguments>
                                 <argument>-m</argument>
                                 <argument>pylint</argument>
+                                <argument>--max-line-length</argument>
+                                <argument>119</argument>
```
Comment (ahornace) [inline opengrok-tools/pom.xml:284]: Why 119? I thought that the rule is 120 inclusive.

File: opengrok-tools/pom.xml [hunk 2]
```diff
@@ -299,6 +301,8 @@ Portions Copyright (c) 2017-2018, 2020, Chris Fraire <cfraire@me.com>.
                             <arguments>
                                 <argument>-m</argument>
                                 <argument>flake8</argument>
+                                <argument>--max-line-length</argument>
+                                <argument>119</argument>
```
Comment (vladak) [inline opengrok-tools/pom.xml:305]: introduce a variable ?

**Comment by vladak** [comment]
I have no problem with this. This should match the style used for Java.

File: opengrok-tools/pom.xml (hunk 1 above)
Comment (tulinkry) [reply opengrok-tools/pom.xml:284]: The default was:

```
  --max-line-length n   Maximum allowed line length for the entirety of this run. (Default: 79)
```

so I believe it is 120 including the newline.

File: opengrok-tools/pom.xml (hunk 2 above)
Comment (tulinkry) [reply opengrok-tools/pom.xml:305]: sure

**Pull request merged**